src/source/YamlGenerator.scala

```diff
@@ -103,11 +103,12 @@ class YamlGenerator(spec: Spec) extends Generator(spec) {
         }.mkString(" deriving(", ", ", ")")
       }
     }
-    td.body match {
-      case i: Interface => "interface" + ext(i.ext)
-      case r: Record => "record" + ext(r.ext) + deriving(r)
-      case e: Enum => "enum"
-    }
+  	td.body match {
```

Comment:
I'm still seeing tabs in the new version of this code, vs. the spaces in the old version.  Needs to be fixed to consistently use spaces (the preference for this repo).